pkg/model/postgresql_service.go

```diff
@@ -12,16 +12,21 @@ import (
 
 func getSpec(dbSecret *v1.Secret, serviceTypeExternal bool) v1.ServiceSpec {
 	spec := v1.ServiceSpec{}
+	isIPAddress := dbSecret != nil && dbSecret.Data[DatabaseSecretExternalAddressProperty] != nil && IsIP(dbSecret.Data[DatabaseSecretExternalAddressProperty])
```

Comment:
Maybe I'd just add a new param to this function instead of calculating `isIPAddress` here. We already know this in `keycloak_reconciler`. But it's just a nitpick, fix it only if you want. :)

Reply:
The idea of putting this inside the method and no adding extra args is that it can do the functionality without extra params, so it has all the information. If we add another param we , by design, could have inconsistency.